src/Protocol/DFRN.php

```diff
@@ -1291,31 +1294,14 @@ public static function deliver($owner, $contact, $atom, $dissolve = false)
 
 			switch ($rino_remote_version) {
 				case 1:
-					// Deprecated rino version!
+				case 2:
```

Comment:
I guess you mustn't handle "2" here.

Reply:
After review, it appears you are right. The handshake with `dfrn_notify` apparently already takes care of agreeing on a specific RINO version.